File: Java/1-50/13.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        String[] inputs = reader.readLine().split(" ");
        int n = Integer.parseInt(inputs[0]);
        int k = Integer.parseInt(inputs[1]);

        if (n == 0) {
            System.out.println("1");
        } else {
            System.out.println(k + 1);
        }
    }
}

File: Kotlin/1-50/13.kt
import java.io.BufferedReader
import java.io.InputStreamReader

fun main() {
    val reader = BufferedReader(InputStreamReader(System.`in`))
    val inputs = reader.readLine().split(" ")
    val n = inputs[0].toInt()
    val k = inputs[1].toInt()

    if (n == 0) {
        println("1")
    } else {
        println(k + 1)
    }
}